npm.js

```diff
@@ -62,12 +62,16 @@ exports.translate = function(load){
 				packages[pkg.name+"@"+pkg.version] = true;
 			}
 		});
-		var configDependencies = ['@loader','npm-extension'].concat(configDeps.call(loader, pkg));
+		var configDependencies = [
+			'@loader',
+			'npm-extension',
+			'process'
```

Comment:
Okay got it. So we need to find another way to load `process`. 

> Because it's almost 150 LOC that would be included always (including production builds) even > in projects that don't need it.

But they're 150 really really tiny lines of code.